enums.py

```diff
@@ -422,3 +422,12 @@ class TaskType(Enum):
     INFORMATION_SOURCE = "information_source"
     UPLOAD_TASK = "upload"
     WEAK_SUPERVISION = "weak_supervision"
+
+class AgreementType(Enum):
+    EMBEDDING = "emedding"
+
+class EmbeddingPlatform(Enum):
+    PYTHON = "python"
+    HUGGINGFACE = "huggingface"
+    COHERE = "cohere"
+    OPENAI = "openai"
```

Comment:
why are these in lowercase? Normally we use uppercase values (with a few exceptions like table names)

Reply:
Makes sense. Agreement type is new though

Reply:
Changed embedding to EMBEDDING for agreement type